Makefile

```diff
@@ -270,6 +275,10 @@ golangci-lint: $(LOCALBIN)
 	@test -x $(GOLANGCI_LINT) && $(GOLANGCI_LINT) version | grep -q $(GOLANGCI_LINT_VERSION) || \
 	GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
 
+.PHONY: nilaway
+nilaway: $(LOCALBIN)
+	@test -x $(NILAWAY_LINT) || GOBIN=$(LOCALBIN) go install go.uber.org/nilaway/cmd/nilaway@latest
```

Comment:
I've checked it.
It's so sad that we can't use particular version ;(

Reply:
yes, they don't have releases